models/bamf_nnunet_ct_lung/config/default.yml

```diff
@@ -0,0 +1,44 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet Lung and Nodules segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- module: NNUnetRunner
+  nnunet_task: Task777_CT_Nodules
+  roi: LUNG+NODULE
+- module: NNUnetRunner
+  nnunet_task: Task775_CT_NSCLC_RG
+  roi: LUNG,LUNG+FDG_AVID_TUMOR
+- LungPostProcessor
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  NNUnetRunner:
+    in_data: nifti:mod=ct
+    nnunet_model: 3d_fullres
+    export_prob_maps: False
+
+  LungPostProcessor:
+    in_rg_data: nifti:mod=seg:nnunet_task=Task775_CT_NSCLC_RG
+    in_nodules_data: nifti:mod=seg:nnunet_task=Task777_CT_Nodules
+
+  DsegConverter:
+    model_name: Bamf CT Lung and Nodule Segmentation
+    target_dicom: dicom:mod=ct
+    source_segs: nifti:mod=seg:processor=bamf
+    skip_empty_slices: True
+
+  DataOrganizer:
+    targets:
+    - dicomseg-->[i:sid]/bamf_nnunet_ct_lung.seg.dcm
```

Comment:
It seems that `bamf_nnunet_ct_lungnodules` would be a better fit for a name?

Reply:
I have updated with this naming convention.

Reply:
thx! This would require to change the folder name (`modles/$model_name`) too.

Reply:
updated the model directory to `bamf_nnunet_ct_lungnodules`